web-new/src/components/MevExtractionCard.tsx

```diff
@@ -0,0 +1,9 @@
+
+interface MevExtractionCardProps {
+    children: React.ReactNode
+}
+
+export default function MevExtractionCard(props: MevExtractionCardProps) {
+    const {children} = props;
+    return <div className="flex-1 flex overflow-hidden flex-col p-4 uppercase leading-none bg-brand-60">{children}</div>
```

Comment:
please follow 2 space padding as rest of code

Reply:
Sure thing!